Employment Policies/Working Remotely.md

```diff
@@ -1,11 +1,91 @@
+This policy is written in the context of our current company setup, which is that the majority of employees work in the same office for the most of the work week. Specifically, this document addresses working from home on a regular basis and/or working remotely for up to one week out of any given month. It does not address:
 
-Employees with chronic or terminal illnesses should talk with the founder they report to about their needs for remote work, flexible time, and other support.
+* A fully remote employee
+* An employee who works from the office 1wk/month or is remote for the majority of their time
+* A team that is distributed
 
-As we grow, we hope to add the capability for employees to work from home or work remotely more consistently, but, at our current size, the in-person interactions in the office are too critical to pass up.
+If and when the current company setup changes, we should address the above situations. For the purpose of this document, we interpret remote work as as a co-located employee of Clef working from home or from an irregular-place. 
 
-When we reach 10 employees, we will review this policy to see if our capabilities to support remote work have expanded.
+## Approach
+
+Fundamentally, working remotely is a privilege that comes with both benefits and costs. We should treat it as such.
+
+The benefits are primarily ones of flexibility: as individuals, we are able to work when we want to and where we want to. This lets us do our best work and contribute to our team in a custom-crafted framework that lets us lead more fulfilling lives. The costs are primarily ones of communication: being out of the office means more base-line work and incidental complexity for all employees. More is required of both the remote employee and the employees in the office to keep in sync, collaborate, and own the work we do in a visible way. 
+
+When we work remotely, it's our responsibility to accept the benefits graciously, and go above and beyond to not let the costs affect our work or the work of our teammates. A remote workday:
+
+* is not shorter than a day at the office
+* is not less productive
+* is not an excuse for missing commitments (be they meetings or deadlines). 
+
+Even when we aren't in the office, our teammates are still counting on us and it is our responsibility to make sure we are exceeding their expectations.
+
+At the same time, it is the responsibility of workers in the office to reasonably accommodate remote workers. This means being willing to schedule meetings in light of timezone differences and to manage expectations around immediate feedback.  
+
+The guidelines in this document (and the handbook at large) give us tools to make remote work easier, but tooling is only a small part of the solution. All employees need to complement these tools with the extra work required to make being out of the office an effective thing for both the individual remote worker and the team of people who rely on them.
```

Comment:
Clarity: last sentence, how about "Every employee needs to complement these tools with _extra_ work to make remote work effective for the employee working remotely and the rest of our team."